Liturgy/C-TheSecondAntiphon/GNE-OnlyBegottenSon.ly

```diff
@@ -17,29 +17,78 @@ recite = \once \override LyricText.self-alignment-X = #-1
 \defineBarLine "invisible" #'("" "" "")
 global = {
   \time 1/1 % Not used, Time_signature_engraver is removed from layout
-  \key g \major
+  \key f \major
   \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
 }
 
 verseOne = \lyricmode {
-
+  Glo -- \recite "ry to the Father and to the Son and to the" Ho -- ly Spi -- rit,
+  \recite "Both now and ever and unto a" ges of a -- ges. A -- men.
+  On -- ly -- be -- got -- ten Son and Im -- mor -- tal Word of God,
+  who for our sal -- va -- tion willed to be in -- car -- nate
+  of the Ho -- ly The -- o -- to -- kos and Ev -- er -- Vir -- gin Ma -- ry,
+  who with -- out change be -- came man and was cru -- ci -- fied,
+  who is One of the Ho -- ly Tri -- ni -- ty;
+  glo -- ri -- fied with the Fa -- ther and the Ho -- ly Spi -- rit,
+  O Christ, our God, tramp -- ling down death by death, save us.
 }
```

Comment:
I'm pretty sure that the word "who" should be capitalized here, as it refers to Christ.

But also, should we redo this with the Antiochian text/phrasing? Here's an example from http://ww1.antiochian.org/sites/default/files/refrain-troparion_of_the_second_antiphon-byzt2.pdf that shows minor differences, mostly in phrase ordering toward the end:

Only begotten Son and Word of God Who art immortal,
Who for our salvation willed to be incarnate of the Holy Theotokos and Ever-Virgin Mary,
and without change became man;
and was crucified, O Christ our God,
and trampled down death by death;
Who art one of the Holy Trinity,
glorified together with the Father and the Holy Spirit:
save us.

It's probably minor enough that we can leave it for now, but maybe we file an issue on this to get to later.

Reply:
https://github.com/JedBartlett/OrthodoxLiturgics/issues/13